dojo/unittests/test_jira_import_and_pushing_api.py

```diff
@@ -44,8 +44,8 @@ def assert_cassette_played(self):
             self.assertTrue(self.cassette.all_played)
 
     def _get_vcr(self, **kwargs):
-        my_vcr = super(JIRAConfigAndPushTestApi, self)._get_vcr(**kwargs)
-        my_vcr.record_mode = 'once'
+        my_vcr = super(JIRAImportAndPushTestApi, self)._get_vcr(**kwargs)
+        my_vcr.record_mode = 'none'
```

Comment:
Was this change intended?

Reply:
I think none is the right one for running existing unit tests